dts/bindings/spi/gd,gd32-spi.yaml

```diff
@@ -24,3 +24,23 @@ properties:
 
   pinctrl-names:
     required: true
+
+  tx-dma-priority:
+    type: int
+    description: Priority of DMA for transmit
+    default: 0
+
+  rx-dma-priority:
+    type: int
+    description: Priority of DMA for receive
+    default: 0
+
+  tx-dma-peripheral:
```

Comment:
Please refer https://github.com/zephyrproject-rtos/zephyr/pull/42215#discussion_r827790806. 

We need to provide a simple way for the user to select the peripheral easily.

Reply:
This can be done in another PR, not related to this.